go/pkg/fakes/server.go

```diff
@@ -200,6 +200,12 @@ func (e *TestEnv) Set(cmd *command.Command, opt *command.ExecutionOptions, res *
 		ac.Timeout = dpb.New(cmd.Timeout)
 	}
 	acDg = digest.TestNewFromMessage(ac)
+	if ar.StderrDigest != nil {
+		stderrDg = digest.Digest{Hash: ar.StderrDigest.Hash, Size: ar.StderrDigest.SizeBytes}
```

Comment:
There is a convenient function for that: `digest.NewFromProtoUnvalidated(ar.StderrDigest)`.